src/UI/AddMovies/SearchResultCollectionView.js

```diff
@@ -39,7 +39,6 @@ module.exports = Marionette.CollectionView.extend({
 		setExisting : function(tmdbid) {
 				var movies = this.collection.where({ tmdbId : tmdbid });
 				console.warn(movies);
-				//debugger;
 				if (movies.length > 0) {
 						this.children.findByModel(movies[0])._configureTemplateHelpers();
 						//this.children.findByModel(movies[0])._configureTemplateHelpers();
```

Comment:
Should this comment be removed while we are in here? Looks like it’s identical to non commented line above.

Reply:
Well, there are a **lot** of commented lines everywhere. I didn't remove those because I'm assuming most of them are there for a reason and are not leftovers from Sonarr or old commits.